linsolve/tests/benchmark_A_large_shared.py

```diff
@@ -0,0 +1,28 @@
+import linsolve
```

Comment:
I'd be good to say in a comment at the top what these various benchmarks are testing (just to be more explicit than the filenames of the python scripts).